css/style.css

```diff
@@ -181,3 +181,9 @@ footer {
 *::placeholder {
   font-family: "Poppins", sans-serif;
 }
+
+@media screen and (min-width: 992px) and (max-width: 1100px) {
+  .intro {
+    background-size: 100%;
+  }
+}

```

Comment:
- [ ] You can move this `.intro` style to your desktop breaking point and remember to have only one desktop breaking point instead of repeating them over and over.